crypto/dilithium/pqcrystals_dilithium_ref_common/sign.c

```diff
@@ -69,6 +69,19 @@ int crypto_sign_keypair_internal(ml_dsa_params *params,
   /* FIPS 204: line 9 Compute H(rho, t1) and line 10 write secret key */
   shake256(tr, TRBYTES, pk, params->public_key_bytes);
   pack_sk(params, sk, rho, tr, key, &t0, &s1, &s2);
+
+  /* FIPS 204. Section 3.6.3 Destruction of intermediate values. */
+  OPENSSL_cleanse(seedbuf, sizeof(seedbuf));
+  OPENSSL_cleanse(tr, sizeof(tr));
+  OPENSSL_cleanse(&rho, sizeof(rho));
+  OPENSSL_cleanse(&rhoprime, sizeof(rhoprime));
+  OPENSSL_cleanse(&key, sizeof(key));
```

Comment:
Pointer values are no more sensitive than integer offsets. Don't think we need these anymore.